src/ux/ux.ts

```diff
@@ -72,12 +73,21 @@ export class Ux extends UxBase {
   /**
    * Display a table to the console. This will be automatically suppressed if output is disabled.
    *
-   * @param data Data to be displayed
-   * @param columns Columns to display the data in
-   * @param options Options for how the table should be displayed
+   * @param options Table properties
    */
-  public table<T extends Ux.Table.Data>(data: T[], columns: Ux.Table.Columns<T>, options?: Ux.Table.Options): void {
-    this.maybeNoop(() => table(data, columns, { 'no-truncate': true, ...options }));
+  public table<T extends Record<string, unknown>>(options: TableOptions<T>): void {
+    this.maybeNoop(() =>
+      printTable({
+        ...options,
+        // Don't allow anyone to override these properties
+        borderStyle: 'headers-only-with-underline',
```

Comment:
```suggestion
        borderStyle: 'vertical-with-outline',
```

I am fine if we put this to a vote with the team. I think this style makes the tables easier to read.